src/components/media-views.js

```diff
@@ -361,6 +361,7 @@ AFRAME.registerComponent("media-video", {
 
     NAF.utils.getNetworkedEntity(this.el).then(networkedEl => {
       this.networkedEl = networkedEl;
+      this.networkedEl.components.networked.applyPersistentFirstSync();
```

Comment:
as noted in the other PR, any reason we can't push this policy into NAF itself?

Reply:
We can't apply the stashed sync in `init` on `networked`, the rest of the entity's components are not initialized at that point. If we did apply it then, it would result in those components being created and then overwritten by the rest of Hub's scene and media initialization process.

Reply:
In particular, we have to apply the first sync after component inits, but before we do this very next bit, where we try to take ownership of scene-owned media.

Reply:
ah interesting. i guess i can't come up with a better idea. renaming the function to be less implementation-centric would be nice, since it's a public API, but i can't think of a much better name. eg `notifyPeristentEntityReady` or something

Reply:
Hmm, actually I think I would like to keep the name explicit since I want it to be clear to us when reading this code.